comet/services/torrent_manager.py

```
@@ -485,6 +511,60 @@ def _determine_conflict_key(season, episode) -> str:
+                    )
+                    for row in ordered_rows:
+                        lock_key = row.get("lock_key")
+                        if lock_key is None:
+                            continue
+                        await database.execute(
+                            "SELECT pg_advisory_xact_lock(CAST(:lock_key AS BIGINT))",
+                            {"lock_key": lock_key},
+                        )
+
+                await database.execute_many(query, sanitized_rows)
+            return
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)